src/renderer/components/dialogs/MessageDetail/ReadReceipts.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useEffect, useState } from 'react'
+import { BackendRemote } from '../../../backend-com'
+import { T } from '@deltachat/jsonrpc-client'
+import { selectedAccountId } from '../../../ScreenController'
+
+import styles from './styles.module.scss'
+import { Avatar } from '../../Avatar'
+import { InlineVerifiedIcon } from '../../VerifiedIcon'
+import moment from 'moment'
+import useTranslationFunction from '../../../hooks/useTranslationFunction'
+type ReadReceiptsListProps = { messageId: number }
+
+export function ReadReceiptsList(props: ReadReceiptsListProps) {
+  const tx = useTranslationFunction()
+  const accountId = selectedAccountId()
+  const [receipts, setReceipts] = useState<T.MessageReadReceipt[]>([])
+  useEffect(() => {
+    BackendRemote.rpc
+      .getMessageReadReceipts(accountId, props.messageId)
+      .then(setReceipts)
+  }, [props.messageId, accountId])
```

Comment:
Also this doesn't react to new read receipts after render? Would it be hard to add?

Reply:
the dialog could add an event listener for read message that compares the message id and calls a refresh function. I can do a meeting with @meganoahj to teach him how.